docs/ref/csrf.txt

```diff
@@ -137,17 +137,49 @@ and read the token from the DOM with JavaScript:
 
     {% csrf_token %}
     <script type="text/javascript">
```

Comment:
Missing src attribute

Reply:
@graingert This `script` element contains embedded script so it callot also have a reference.

> If a script element has a src attribute specified, it should not have a script embedded inside its tags since it can lead to unexpected behavior. ...

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-src

Reply:
It shouldn't contain a body